packages/@lwc/engine/src/framework/hooks.ts

```diff
@@ -198,6 +198,7 @@ export function createViewModelHook(vnode: VCustomElement) {
         mode,
         owner,
         isRoot: false,
+        renderer: owner.renderer,
```

Comment:
I suspect that this can't be get from owner directly during `createVM` because there are cases in which there is no owner! Is this accurate to say?

Reply:
I far as I understand the only case where there is no owner is when creating a new component tree using `createElement`. Since `createElement` is part of the dom package, we can inject the renderer from there.